lib/events/codes.go

```diff
@@ -346,10 +349,16 @@ const (
 	TrustedClusterCreateCode = "T7000I"
 	// TrustedClusterDeleteCode is the event code for removing a trusted cluster.
 	TrustedClusterDeleteCode = "T7001I"
-	// TrustedClusterTokenCreateCode is the event code for
-	// creating new join token for a trusted cluster.
+	// TrustedClusterTokenCreateCode is the event code for creating new
+	// provisioning token for a trusted cluster. Deprecated in favor of
+	// [ProvisionTokenCreateEvent].
 	TrustedClusterTokenCreateCode = "T7002I"
 
+	// ProvisionTokenCreateCode is the event code for creating a provisioning
+	// token, also known as Join Token. See
+	// [github.com/gravitational/teleport/api/types.ProvisionToken].
```

Comment:
Nit: I don't see this pattern of reference being used commonly elsewhere, I think finding the usage of this variable in the codebase is self explanatory.

```suggestion
	// ProvisionTokenCreateCode is the event code for creating a provisioning
	// token, also known as Join Token.
```